config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -2117,6 +2117,10 @@
 # DEFAULT: true (reminder will be checkmarked when creating new assignment)
 # asn.reminder.email.default=false
 
+# S2U-30 Assignments will be assigned by default to the instructors' groups.
+# DEFAULT: TRUE (Enabled, assigned to the instructor groups and not the site)
+# assignments.assign.to.instructor.groups=false
```

Comment:
This is still in the wrong section, place it at the end of the assignments section, it could be different in 22.x but exists:

https://github.com/sakaiproject/sakai/blob/s2u-22x/config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties#L2089